CONTRIBUTING.md

```diff
@@ -1,5 +1,80 @@
 # Contributing
 
-## Adding a New Lint or Verification
+## Adding a New Lint
```

Comment:
It seems a bit weird to me to add this much code to the markdown file. Like if you later find a bug or refactor, then you also need to change this markdown file, right? But I think this is ok.